### DISCUSSION TITLE
Install git for windows.vfs Use pacman

### DISCUSSION THREAD

**Discussion by kouhe3** [microsoft/git#572] [Q&A]
https://github.com/git-for-windows/git/wiki/Install-inside-MSYS2-proper

↑ Install normal Git for Windows in MSYS2.

Can we Install git for win *vfs* version in MSYS2?

**Comment by derrickstolee** [comment]
Since that method is not officially supported by Git for Windows, I can guarantee that we will not support it in this fork. I'm sorry if this blocks a specific workflow, but it doesn't seem to meet our bar for working on it.
